terraform/development/shared/main.tf

```diff
@@ -153,6 +153,16 @@ module "database" {
   vpc_id                      = module.networking.vpc_id
 }
 
+module "deployment_role" {

```

Comment:
This might at some point extend to be used for more automated terraform deployments, but for now it's only set up for allowing the review app ones.